site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/converter/OrganizationConverter.java

```diff
@@ -30,6 +30,9 @@
             fhirOrganization.setId(new IdType(input.getSiteId()));
         }
         setParentOrganization(input, fhirOrganization);
+        if (input.getSiteType() != null) {
+            fhirOrganization.setImplicitRules(input.getSiteType());
```

Comment:
Has this been tested against the FHIR server? The implictRules attribute has to be a valid URI - this is fairly loose, but I believe has to start with a scheme identifier (e.g. we could use sitetype:CCO)

Reply:
I have tested this against FHIR and it seems to hold